src/CommunityToolkit.Maui.MediaElement/Services/MediaControlsService.android.cs

```diff
@@ -0,0 +1,317 @@
+
+		var play = new Intent(this, typeof(MediaControlsService));
+		play.SetAction("MediaAction.play");
+		var pPlay = PendingIntent.GetService(this, 1, play, PendingIntentFlags.UpdateCurrent);
+		actionPlay ??= new NotificationCompat.Action.Builder(Resource.Drawable.exo_controls_play, ACTION_PLAY, pPlay).Build();
+
+		var previous = new Intent(this, typeof(MediaControlsService));
+		previous.SetAction("MediaAction.previous");
+		var pPrevious = PendingIntent.GetService(this, 1, previous, PendingIntentFlags.UpdateCurrent);
+		actionPrevious ??= new NotificationCompat.Action.Builder(Resource.Drawable.exo_controls_rewind, ACTION_BACK, pPrevious).Build();
+
+		var next = new Intent(this, typeof(MediaControlsService));
+		next.SetAction("MediaAction.next");
+		var pNext = PendingIntent.GetService(this, 1, next, PendingIntentFlags.UpdateCurrent);
+		actionNext ??= new NotificationCompat.Action.Builder(Resource.Drawable.exo_controls_fastforward, ACTION_FORWARD, pNext).Build();
+		if (!notificationExists)
+		{
+			notification?.AddAction(actionPrevious);
+			notification?.AddAction(actionPause);
+			notification?.AddAction(actionNext);
+		}
+	}
+
+	void BroadcastUpdate(string action)
+	{
+		if (Build.VERSION.SdkInt >= BuildVersionCodes.Tiramisu)
+		{
+			return;
+		}
+		var intent = new Intent(ACTION_UPDATE_UI);
+		intent.PutExtra("ACTION", action);
+		LocalBroadcastManager.GetInstance(this).SendBroadcast(intent);
+	}
+
+	protected override void Dispose(bool disposing)
+	{
+		if (!disposedValue)
+		{
+			if (disposing)
+			{
+				safeHandle?.Dispose();
+				safeHandle = null;
+			}
+			_ = (audioManager?.AbandonAudioFocus(null));
+			audioManager?.SetParameters("Ducking=false");
+			audioManager?.Dispose();
+			mediaSession?.Release();
+			mediaSession?.Dispose();
+			mediaSession = null;
+			
+			if (receiveUpdates is not null)
+			{
+			
+				receiveUpdates.PropertyChanged -= ReceiveUpdates_PropertyChanged;	LocalBroadcastManager.GetInstance(Platform.AppContext).UnregisterReceiver(receiveUpdates);
+			}
+			receiveUpdates?.Dispose();
+			receiveUpdates = null;
+			disposedValue = true;
+		}
+		base.Dispose(disposing);
+	}
+}
+
+/// <summary>
+/// A <see cref="BroadcastReceiver"/> that listens for updates from the <see cref="MediaManager"/>. 
+/// </summary>
+class ReceiveUpdates : BroadcastReceiver
+{
+	string action = string.Empty;
+	public string Action
+	{
+		get => action;
+		set
+		{
+			if (action != value)
+			{
+				action = value;
+				PropertyChanged?.Invoke(this, new PropertyChangedEventArgs(nameof(Action)));
+			}
+		}
+	}
+	
+	public event PropertyChangedEventHandler? PropertyChanged;
+
+	/// <summary>
+	/// Method that is called when a broadcast is received.
+	/// </summary>
+	/// <param name="context"></param>
+	/// <param name="intent"></param>
+	public override void OnReceive(Context? context, Intent? intent)
+	{
+		Action = intent?.GetStringExtra("ACTION") ?? string.Empty;
+	}
+	
+	public string GetAction()
+	{
+		return Action;
+	}
```

Comment:
We already have the `get` on the `Action` property, no need fo this extra method.
```suggestion
```